CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Use the default match and search option in Advanced search when they are not entered. Refs STSMACOM-793.
 * Show successful toast notifications for Create and Edit actions in `<ControlledVocab>`. Refs STSMACOM-796.
 * `<ControlledVocab>` - last updated by column - show "System" when items are created by system user. Refs STSMACOM-797.
+* Add field type `DATE_PICKER` to custom fields components. Refs FCFIELDS-46.
```

Comment:
Please move the original Jira ticket to STSMACOM project and update the ticket number here
No need to re-create the PR, after merging the branch should be deleted anyways
We only reference tickets from corresponding Jira projects in repositories so we can correctly track release versions.
The next released version of `stripes-smart-components` will be `9.1.0` or `10.0.0`, but FCFIELDS project might have a different next release version, and there could be a lot of confusion on what has been released, where and when

Reply:
Done.